composer.json

```diff
@@ -35,5 +35,7 @@
         "psr-4": {
             "SimpleBus\\JMSSerializerBundleBridge\\Tests\\": "tests"
         }
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

Comment:
Since SF 4.0 is released, can we drop this?

Reply:
We can, but I think we can keep it to easily test with version of dependencies not yet released to prepare necessary changes for compatibility.

Reply:
I just removed these lines in https://github.com/SimpleBus/SymfonyBridge/pull/62 as these seems to create a memory problem in composer in php 5.6